man/SAPHanaSR.py.7

```diff
@@ -272,7 +272,7 @@ the sudo permissions configuration
 path to HANA tracefiles
 .TP
 /usr/sap/$SID/HDB$nr/.crm_attribute.$SITE
-the internal cache for srHook status changes while Linux cluster is down, file is owned by ${SID}adm and must never be touched
+the internal cache for srHook status changes while Linux cluster is down, file is owned and r/w by ${SID}adm and must never be touched
```

Comment:
Please also add the needed permissions to the directory (to CREATE) the needed file.